cmd/benchmark.go

```diff
@@ -0,0 +1,26 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	t "github.com/coinbase/rosetta-cli/pkg/tester"
+	"github.com/spf13/cobra"
+)
+
+var (
+	checkBenchmarkCmd = &cobra.Command{
+		Use:   "check:benchmark",
+		Short: "Benchmark performance of time critical methods of rosetta-cli and rosetta-sdk",
+		Long: `This command can be used to benchmark the performance of time critical methods for rosetta-cli and rosetta-sdk.
+This is useful for optimizing the performance of commands like check:data and check:construction, which can take a long time to complete.
+The performance improvement can be objectively compared.`,
```

Comment:
- "Time-critical" needs a dash.
- Is the correct term `rosetta-sdk` or `rosetta-sdk-go`?

```suggestion
		Long: `This command can be used to benchmark the performance of time-critical methods for rosetta-cli and rosetta-sdk.
This is useful for optimizing the performance of commands like check:data and check:construction, which can take a long time to complete.
The performance improvement can be objectively compared.`,
```

"The performance improvement can be objectively compared." - What does this mean? How would a user compare the improvement? Or is that something that automatically happens once a user runs check:benchmark?